velox/exec/tests/HashJoinTest.cpp

```diff
@@ -33,6 +34,8 @@
 #include "velox/exec/tests/utils/VectorTestUtil.h"
 #include "velox/vector/fuzzer/VectorFuzzer.h"
 
+DECLARE_bool(velox_testing_enable_arbitration);
```

Comment:
No need velox_testing_enable_arbitration is declared in operator test base header file.